src/components/DocumentRenderer/ValidityBanner/ValidityBannerContent.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FunctionComponent } from "react";
+import { CheckStatus } from "./types";
+import { View } from "react-native";
+
+interface ValidityBannerContent {
+  checkStatus?: CheckStatus;
+  isExpanded?: boolean;
+}
+
+export const ValidityBannerContent: FunctionComponent<ValidityBannerContent> = ({
+  checkStatus = "checking",
+  isExpanded = false,
+  children
+}) => {
+  let backgroundColor;
+  switch (checkStatus) {
+    case "valid":
+      backgroundColor = "#EFFAF4";
```

Comment:
Want to put these colors out in the color, easier if we want to standardize i think

Reply:
Yea, initially wanted to push this first, then refactor in another PR, but i'll move it somewhere first.